scripts/tests/test_render_snyk_report.py

```
@@ -0,0 +1,311 @@
+
+    assert counts.critical == 0
+    assert counts.high == 1
+    assert counts.medium == 1
+    assert counts.low == 1
+
+
+def test_status_for():
+    assert report.status_for("Critical", 0) == "✅ Fixed"
+    assert report.status_for("Critical", 1) == "❌ Must fix"
+    assert report.status_for("Medium", 0) == "✅ Fixed"
+    assert report.status_for("Medium", 2) == "ℹ️ Managed Debt"
```

Ruff (0.15.5)

[warning] String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)